src/metatensor/models/cli/train.py

```diff
@@ -351,6 +351,16 @@ def _train_model_hydra(options: DictConfig) -> None:
             )
             validation_datasets.append(validation_dataset)
 
+    if (
+        sum([len(validation_dataset) for validation_dataset in validation_datasets])
```

Comment:
We also need this for the test and the training set, right?

Reply:
Ahh I see. Might it be not enough to change the line above from

```python
if validation_size < 0 or validation_size >= 1:
```

to 

```python
if validation_size <= 0 or validation_size >= 1:
```

?